contribs/drt/src/main/java/org/matsim/contrib/drt/prebooking/PrebookingManager.java

```diff
@@ -163,35 +152,43 @@ private void flushRejections(double now) {
 	// collects new bookings that need to be submitted
 	private final ConcurrentLinkedQueue<PassengerRequest> bookingQueue = new ConcurrentLinkedQueue<>();
 
-	public void prebook(MobsimAgent person, Leg leg, double earliestDepartureTime) {
-		Preconditions.checkArgument(leg.getMode().equals(mode), "Invalid mode for this prebooking manager");
-		Preconditions.checkState(!person.getState().equals(State.ABORT), "Cannot prebook aborted agent");
+	public void prebook(List<PersonLeg> personsLegs, double earliestDepartureTime) {
```

Comment:
Could you please just add one more method prebok(agent, leg, double) that delegates to that onen?